CourseApp/CourseApp/CourseApp.cpp

```diff
@@ -0,0 +1,63 @@
+#include <iostream>
+#include <math.h>
+#include <vector>
+
+double func(const double x)
+{
+    if (abs(x) < 1)
+        return acos(x);
+
+    return pow(1.2, x) - pow(x, 1.2);
+}
+
+std::vector<double> exampleA(const double xStart, const double xEnd, const double xDelta)
```

Comment:
let's return vector of vectors - that will contain both x and y values (only for TaskA and taskB functions)